scripts/perftune.py

```diff
@@ -171,19 +171,19 @@ def restart_irqbalance(banned_irqs):
     perftune_print("Restarting irqbalance: going to ban the following IRQ numbers: {} ...".format(", ".join(banned_irqs_list)))
 
     # Search for the original options line
-    opt_lines = list(filter(lambda line : re.search("^\s*{}".format(options_key), line), cfile_lines))
+    opt_lines = list(filter(lambda line : re.search(r"^\s*{}".format(options_key), line), cfile_lines))
     if not opt_lines:
         new_options = "{}=\"".format(options_key)
     elif len(opt_lines) == 1:
         # cut the last "
-        new_options = re.sub("\"\s*$", "", opt_lines[0].rstrip())
+        new_options = re.sub("\"\\s*$", "", opt_lines[0].rstrip())
         opt_lines = opt_lines[0].strip()
     else:
         raise Exception("Invalid format in {}: more than one lines with {} key".format(config_file, options_key))
 
     for irq in banned_irqs_list:
         # prevent duplicate "ban" entries for the same IRQ
-        patt_str = "\-\-banirq\={}\Z|\-\-banirq\={}\s".format(irq, irq)
+        patt_str = r"\-\-banirq\={}\Z|\-\-banirq\={}\s".format(irq, irq)
```

Comment:
`r"--banirq={}\Z|--banirq={}\s".format(irq, irq)`

Reply:
thank you, Vlad. after a second thought, i went further by replacing `"--banirq"` with a variable.
with my change, it looks like
```py
        opt = f"--banirq={irq}"
        patt_str = rf"{opt}\Z|{opt}\s"
        if not re.search(patt_str, new_options):
            new_options += f" {opt}"
```
does it look better now?

Reply:
It definitely does!
I intentionally didn't suggest you using f-strings in my initial suggestion because there rest of file uses the old-school formatting all other the place.  But if you are up to using them - I won't mind.

Reply:
i see. there are indeed a lot of places in this script, where  we could use f-string for better readability. yeah, i defend my changes with "readability" quite often =) but i hold my hands back when they tried to do the refactor. as, it might be better to change it piecemeal for reducing the risk of change, and reducing review's workload for reviewing each PR.